_topics/en/silent-payments.md

```diff
@@ -73,5 +73,23 @@ excerpt: >
 
 ---
 
+Traditionally, a new address should be generated whenever a user receives
+two or more payments. This is because when you receive several payments
+to the same Bitcoin address, others can reasonably assume that the same
+person received all of those payments even if the payments are later
+spent in separate transactions. This is also known as address reuse,
+and more can be found under [output linking][topic output linking].
+
+With silent payments, a receiver can generate a silent payment address
+and make it publicly known. The sender then takes a public key from
+one of their chosen inputs for the payment, and uses it to derive a
+shared secret which is then used to tweak the static silent payment address,
+before sending the payment to the tweaked address.
+
+The intended recipient detects the payment by scanning every transaction
+in the blockchain. This is the main downside, along with the requirement to
+control your own input(s) as the sender. An example use case would be
+one-time donations.
```

Comment:
> The intended recipient detects the payment by scanning every transaction
in the blockchain

This isn't quite accurate. The difference is with BIP32 style wallets, the `scriptPubKeys` can be generated ahead of time and then "watched" for to see if they appear in the UTXO set. If you get a match, the wallet then needs to get transaction data from somewhere to add it to the wallet. With silent payments, if you see a taproot output in the UTXO set, you don't know if it is yours or not until you do the ECDH step. ECDH is computationally more expensive than pre-generating `scriptPubKeys` and to do this ECDH you need data from the transaction inputs. 

Concretely, this means a wallet will need to get 33 bytes of data per transaction that it believes is a silent payment transaction (i.e. at least one eligible input and at least one taproot output) and needs to do an ECDH with this 33 bytes.

Sadly, I don't have a suggestion on how to word this succinctly, but maybe something more along the lines of 

>the intended recipient detects the payment by doing ECDH with the summed input public keys of the transaction and the scan key from their address. This is the main downside in that this is computationally more expensive than simply scanning the UTXO set for a `scriptPubKey` and requires access to the summed input public keys."

Regarding the sender:

> along with the requirement to control your own input(s) as the sender

This isn't a requirement for silent payments. Rather, it was designed with collaborative use cases in mind e.g. coinjoins. I think it would be more accurate to say that using silent payments in a collaborative setting is left for future work and that it is still an open question whether or not using silent payments in a collaborative setting is provably secure.

Reply:
Thanks for calling out the difference between silent payments and BIP32 style wallets, the side-by-side comparison is really helpful! It looks like recipients still need to scan eligible transactions -- but instead of looking for an exact UTXO match, they're only looking at the taproot outputs, and do an ECDH computation on each one to check whether the output belongs to them.

I'll update this to include the ECDH computation part you mentioned, as well as the comparison with BIP32.